sdk/eventgrid/azure-eventgrid/azure/eventgrid/__init__.py

```diff
@@ -1,18 +1,23 @@
 # coding=utf-8
 # --------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for
-# license information.
-#
+# Licensed under the MIT License. See License.txt in the project root for license information.
 # Code generated by Microsoft (R) AutoRest Code Generator.
-# Changes may cause incorrect behavior and will be lost if the code is
-# regenerated.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
 
-from .event_grid_client import EventGridClient
+from ._publisher_client import EventGridPublisherClient
+from ._consumer import EventGridConsumer
+from ._helpers import generate_shared_access_signature
+from ._models import CloudEvent, CustomEvent, EventGridEvent, DeserializedEvent
+from .shared_access_signature_credential import EventGridSharedAccessSignatureCredential
 from .version import VERSION
 
-__all__ = ['EventGridClient']
-
+__all__ = ['EventGridPublisherClient', 'EventGridConsumer', 'CloudEvent', 'CustomEvent', 'DeserializedEvent', 'EventGridEvent', 'generate_shared_access_signature', 'EventGridSharedAccessSignatureCredential']
 __version__ = VERSION
 
+try:
```

Comment:
That should not be here, this is intended for autorest generated code, which this file isn't